images.CI/azure-pipelines/image-generation.yml

```diff
@@ -55,6 +55,14 @@ jobs:
                         -ImageName ${{ parameters.image_type }} `
                         -AccessToken $(RELEASE_TARGET_TOKEN)
 
+  - task: PowerShell@2
+    displayName: 'Output Readme file content'
+    inputs:
+      targetType: 'inline'
+      script: |
+        $docsPath = Get-ChildItem -Path "images" -Include ${{ parameters.image_readme_name }} -Recurse -Depth 1 | Select-Object -First 1
```

Comment:
I'm not getting it - why we showing output for Readme file from repo, thats updated after image deployment, instead of showing content for InstalledSoftware.md thats generated with Image?

Reply:
There is no any `InstalledSoftware.md`.
All packer templates just replace existing file in the repository. Please see packer templates for details

Reply:
There is always `InstalledSoftware.md` generated in the produced image root folder. Can't we download it into build agent, right after image generation?

Reply:
And I'm actually don't understand, why we need to show the output for Readme, if we can't get InstalledSoftware, because Readme content always will be outdated, if you compare it with the generated image.

Reply:
We have already do that.
![image](https://user-images.githubusercontent.com/16715858/76503396-e038b800-6456-11ea-82b6-0f58b73efd3e.png)
but they are downloaded to templates directory and this step searches them in this directory

Reply:
Missed that, thanks for the explanation!